build.gradle

```diff
@@ -1,7 +1,64 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.8'
+    }
+}
+
 plugins {
+    id 'idea'
     id 'java'
     id 'maven'
     id 'maven-publish'
+    id 'com.google.protobuf' version '0.8.8'
+}
+
+sourceSets {
+    main {
+        proto {
+            srcDir 'src/main/resources/proto'
+        }
+        java {
+            srcDir 'src/main/java'
+        }
+    }
+    test {
+        proto {
+            srcDir 'src/test/resources/proto'
+        }
+        proto {
```

Comment:
Do we need to change this `proto` to `java`? I'm not sure because I'm not that familiar with gradle :)